### ISSUE TITLE
Data issue in NER training section?

### ISSUE THREAD

**Issue by AngledLuffa** [l3cube-pune/MarathiNLP#1]
At line 159508 of the train_iob.txt file, sentence 17197.0, there is the following:

```
या      O       17197.0
मंत्राची  O       17197.0
देवता    O       17197.0
गणपती   O       17197.0
 ँ       O       17197.0
हा      O       17197.0
तो      O       17197.0
मंत्र     O       17197.0
```

I don't know much about Marathi, to be honest, but to my interpretation this is a Candrabindu mark with no previous character it is marking.  I believe that to be an error.  Would you confirm, and if so, would you suggest a fix?

Thanks!

**Comment by Onkar-2803** [comment]
The '  ँँ  ' symbol seems to be a wrong character. We would update the dataset to rectify it. For now, you can remove the symbol along with it's annotation.

**Comment by AngledLuffa** [comment]
Thanks, sounds good!

**Comment by AngledLuffa** [comment]
I don't see an update to the file when I do a git pull - I think this issue is still here.

**Comment by ravirajoshi** [comment]
Can you check now, fixed with [280fd58](https://github.com/l3cube-pune/MarathiNLP/commit/280fd582fe945222313fc907d218cc735542e48e)

**Comment by AngledLuffa** [comment]
Thanks, that fixes it!
